tests/TestHelper.php

```diff
@@ -85,3 +85,4 @@
 // Suppress DateTime warnings
 date_default_timezone_set(@date_default_timezone_get());
 
+define('PHPUnit_MAIN_METHOD', 'dontuse');
```

Comment:
This prevents any tests that define the static `main` method from taking over the phpunit execution. Ideally these methods (and the blocks of code before and after the test classes that look for this constant) should be removed, but this was the path of least resistance for now.